website/docs/reference/runtime/window.mdx

```diff
@@ -41,6 +41,12 @@ Centers the window on the monitor the window is currently on.
 Go: `WindowCenter(ctx context.Context)`<br/>
 JS: `WindowCenter()`
 
+### WindowExecJS
+
+Executes arbitrary JS code in the window.
```

Comment:
Should we explicitly mention this is fire and forget, so incorrect/failing script snippets won't be logged out somewhere and won't panic?

Reply:
Yes, great point. I'll update the docs.

Reply:
@stffabi - docs updated. Looks like there's a merge conflict caused by #1931 (clashes with the embed commits). Happy to look at it later but if it's obvious to you and a really quick fix then that would be cool 🙏 I think we're almost ready for v2.0.1 🎉

Reply:
Just pushed the fix for the conflicts.

Reply:
Docs looking great 👍